### DISCUSSION TITLE
2024-05-15

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#874] [中文社区双周会]
上次双周会记录 https://github.com/ant-design/ant-design-web3/discussions/822

**Comment by yutingzhao1991** [comment]
讨论下 Ant Design meetup 的事情 https://github.com/ant-design/ant-design-web3/discussions/875 看一下有没有社区同学想要来分享的

**Comment by yutingzhao1991** [comment]
几个关键事情的进展：
- Payment 组件
- 官网 UI 升级

**Reply by kiner-tang** [reply]
官网 UI 升级具体有啥工作呀

**Reply by yutingzhao1991** [reply]
主要是首页 UI 做了下优化，有兴趣看看吗

**Reply by kiner-tang** [reply]
可以，如果时间充裕的话，这边可以用碎片时间搞一下。到时候拆分一下子任务看看。

**Comment by yutingzhao1991** [comment]
Wtfswap 的设计稿评审

**Comment by jeasonstudio** [comment]
在途的 PR 过一下：
* https://github.com/ant-design/ant-design-web3/pull/772 @jeasonstudio 
* https://github.com/ant-design/ant-design-web3/pull/873 @AmAzing129

**Comment by jeasonstudio** [comment]
Batch Money DApp 改造排期

https://www.batch.money/ 是一个分账 DApp，来自 WTF 社区，类似的产品：
* https://mct.xyz/token-sender
* https://disperse.app/

**Comment by AmAzing129** [comment]
Coinbase 推的 smart wallet 可以跟进一下，看看有什么可以做的 @jeasonstudio 

- https://demo.dynamic.xyz/?use-environment=Coinbase

- https://x.com/coinbasewallet/status/1790488024976883973?s=46
